### PR TITLE
LPS-168113 Created Button Action “Clear”

### PR DESCRIPTION
**Pull request by samuelleolopes**

### CODE REVIEW

**Comment by samuelleolopes** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-168113 [FE] Create Button Action “Clear”</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-168113">[FE] Create Button Action “Clear”</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Progress</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Samuel Lopes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Samuel Lopes</td>
        </tr>
                </table>
          </details><details><summary>More details about: <b>LPS-168112 [FE] Create Link From Floating Box to Table</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-168112">[FE] Create Link From Floating Box to Table</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Progress</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Samuel Lopes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Samuel Lopes</td>
        </tr>
                </table>
          </details><details><summary>More details about: <b>LPS-168111 [FE] Add Floating Box Component on Subtask Page</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-168111">[FE] Add Floating Box Component on Subtask Page</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Minor</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Progress</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Samuel Lopes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Samuel Lopes</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

2 files changed, 27 insertions(+), 1 deletion(-)

**Comment by samuelleolopes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx [hunk 1]
```diff
@@ -124,7 +126,28 @@ const SubtasksCaseResults = () => {
...
+						primaryButtonProps={{
+							title: i18n.translate('split-tests'),
+						}}
+						selectedCount={selectedRows.length}
+						tooltipText={i18n.translate(
+							'move-selected-tests-to-a-new-subtask'
+						)}
+					/>
+				);
+			}}
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx:149]: Simplify

```suggestion
			{(_, {dispatch, listViewContext: {selectedRows}}) => (
					<FloatingBox
						clearList={() =>
							dispatch({
								payload: [],
								type: ListViewTypes.SET_CHECKED_ROW,
							})
						}
						isVisible={!!selectedRows.length}
						primaryButtonProps={{
							title: i18n.translate('split-tests'),
						}}
						selectedCount={selectedRows.length}
						tooltipText={i18n.translate(
							'move-selected-tests-to-a-new-subtask'
						)}
					/>
				)
			}
```

**Review by welitonpe — COMMENTED** [review]
Simplify

**Comment by samuelleolopes** [comment]
ci:test:sf

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**